enterprise_learner_portal/api/v1/views.py

```diff
@@ -55,16 +42,50 @@ def get(self, request):
             user_id=user.id,
             enterprise_customer__uuid=enterprise_customer_id,
         )
-        course_ids_for_ent_enrollments = EnterpriseCourseEnrollment.objects.filter(
+        enterprise_enrollments = EnterpriseCourseEnrollment.objects.filter(
             enterprise_customer_user=enterprise_customer_user
-        ).values_list('course_id', flat=True)
-
-        overviews = get_course_overviews(course_ids_for_ent_enrollments)
```

Comment:
Maybe just move the `get_course_overviews` call to the serializer, but keep the ability to pass multiple enterprise enrollments to the serializer.

Reply:
By default the serializer will only ever work on 1 object at a time though, so even if we hand multiple course enrollments to the serializer, still would be doing a `get_course_overviews` call for each item in that list/queryset we hand it

A suggestion: in the view, make 1 call to get all the course overviews that we will ever need to look at, and then pass that to the serializer's context. So instead of calling `get_course_overviews` in the serializer,  you could just lookup the course_overview info in the dict that exists in the context